demos/demo-arm64.js

```diff
@@ -0,0 +1,60 @@
+var e = new uc.Unicorn(uc.ARCH_ARM64, uc.MODE_ARM64);
+var a = new ks.Keystone(ks.ARCH_ARM64); // , ks.MODE_ARM);
+var d = new cs.Capstone(cs.ARCH_ARM64, cs.MODE_ARM);
+
+// Instruction Pointer
+function pcRead() {
+    return e.reg_read_i64(uc.ARM64_REG_PC);
+}
+function pcWrite(value) {
+    return e.reg_write_i64(uc.ARM64_REG_PC, value);
```

Comment:
This fails for `value >= (1 << 52)`, we need to add another method:
```js
this.reg_write_i64    = function (regid, value_lo, value_hi) { /* ... */ };
```
in: https://github.com/AlexAltea/unicorn.js/blob/dd2972c/src/unicorn-wrapper.js#L359